tests/regression/inciter/compflow/Euler/GaussHump/gauss_hump_pdg.q

```diff
@@ -26,7 +26,7 @@ inciter
       sideset 2 end
     end
     bc_outlet
-      p_farfield 1.0
+      farfield_pressure 1.0
```

Comment:
This of course also changes, since I renamed the keyword. Also, this is now optional, because the default is 1.0 but I just kept it.